src/fava/json_api.py

```diff
@@ -288,6 +288,13 @@ def put_source_slice(entry_hash: str, source: str, sha256sum: str) -> str:
     return g.ledger.file.save_entry_slice(entry_hash, source, sha256sum)
 
 
+@api_endpoint
```

Comment:
Could you also add a test for this API endpoint?

Reply:
I added a test for the delete endpoint, but it just uses the internal file api to re-add the deleted entry